config/redirects

```diff
@@ -87,3 +87,37 @@ raw: ${prefix}/manage-apps/configure/config/sync -> ${base}/reference/config/syn
 raw: ${prefix}/manage-apps/configure/config/template-expansions -> ${base}/reference/config/template-expansions/
 raw: ${prefix}/manage-apps/configure/config/triggers -> ${base}/reference/config/triggers/
 raw: ${prefix}/manage-apps/configure/config/values -> ${base}/reference/config/values/
+
+# (DOCSP-26916) [Section] Develop & Deploy Apps
```

Comment:
why have all these redirects if some will go away in the final merge to master?

Reply:
This way we don't miss any!